server/bundler/plugin.js

```diff
@@ -30,11 +31,12 @@ ChunkFileNames.prototype.apply = function( compiler ) {
 						"script.onerror = script.onload = script.onreadystatechange = null;",
 						"delete installedChunks[ chunkId ];",
 						"window.__chunkErrors[ " + JSON.stringify( chunkMaps.name ) + "[chunkId]||chunkId ]=new Error();",
-						"callback.call( null, " + this.requireFn + ")"
+						"return Promise().resolve();"
```

Comment:
Oops, nope. Fixed in de3c00f8d along with some more tidying. I wonder if we should just convert this unreadable nonsense to template literal syntax.